common/source/docs/common-rfd900.rst

```diff
@@ -11,7 +11,7 @@ designed for long range serial communication.  Ranges of `15km and much higher h
 
    These modems are high-power RF devices. It is suggested to
    provide separate +5 voltage to the modems. It is NOT suggested to power
-   these modems directly from Autopilots telemetry port
+   these modems directly from Autopilots telemetry port. This is made simple using an `RFD Adapater Board <https://baskaerospace.com.au/store/telemetry/aerolink-rfd-adapter-board/>`__ from Bask Aerospace.
```

Comment:
could you move this in Bask Aerospace section ? thanks

Reply:
Sure. I've just commited the update.